### PR TITLE
[Fix] Simplify Frame and Layout properties SDK Calls

### PR DESCRIPTION
**Pull request by rizikolik**
This PR removes getFrame/Layout PropertyCalculatedValue function and tests cases related with it.Also makes set frame and set layout functions to accept 2 parameter (frame|layoutId,value)

## Related tickets

-   [WRS-555](https://support.chili-publish.com/projects/WRS/issues/WRS-555)

### CODE REVIEW

File: src/controllers/FrameController.ts [hunk 1]
```diff
@@ -130,87 +130,84 @@ export class FrameController {
...
-        return res.setFrameRotation(selectedFrame?.frameId, parseFloat(calc.toString()));
+        return res.setFrameRotation(frameId, parseFloat(calc.toString()));
     };
 
     /**
      * This method will set the width of a specific frame
-     * @param value The string value that will be calculated (f.e. 1+1 will reult in 2). The notation is in pixels
-     * @param selectedFrame The frame that is selected with all it's properties
+     * @param frameId The ID of a specific frame
+     * @param value The string value that will be calculated (f.e. 1+1 will reult in 2) The notation is in pixels
```
Comment (alexandraFlavia9) [inline src/controllers/FrameController.ts:164]: will result

File: src/controllers/FrameController.ts [hunk 2]
```diff
@@ -130,87 +130,84 @@ export class FrameController {
...
-        return res.setFrameWidth(selectedFrame?.frameId, parseFloat(calc.toString()));
+        return res.setFrameWidth(frameId, parseFloat(calc.toString()));
     };
 
     /**
      * This method will set the x value of a specific frame
-     * @param value The string value that will be calculated (f.e. 1+1 will reult in 2). The notation is in pixels
-     * @param selectedFrame The frame that is selected with all it's properties
+     * @param frameId The ID of a specific frame
+     * @param value The string value that will be calculated (f.e. 1+1 will reult in 2) The notation is in pixels
```
Comment (alexandraFlavia9) [inline src/controllers/FrameController.ts:180]: same as above

**Review by alexandraFlavia9 — APPROVED** [review]

File: src/controllers/LayoutController.ts [hunk 3]
```diff
@@ -121,36 +119,33 @@ export class LayoutController {
 
     /**
      * This method will set the height of the layout to a specific value
+     * @param layoutId The ID of a specific layout
      * @param value The string value that will be calculated (f.e. 1+1 will reult in 2) The notation is in pixels
```
Comment (brapoprod) [inline src/controllers/LayoutController.ts:123]: same as @alexandraFlavia9  already mentioned applies for this file as well

File: src/controllers/FrameController.ts (hunk 1 above)
Comment (rizikolik) [reply src/controllers/FrameController.ts]: Fixed e011559959dd20a563d03bc25ee96d06b7df5991

File: src/controllers/FrameController.ts (hunk 2 above)
Comment (rizikolik) [reply src/controllers/FrameController.ts]: Done e011559959dd20a563d03bc25ee96d06b7df5991

File: src/controllers/LayoutController.ts (hunk 3 above)
Comment (rizikolik) [reply src/controllers/LayoutController.ts]: done e011559959dd20a563d03bc25ee96d06b7df5991

**Review by brapoprod — APPROVED** [review]

**Pull request merged**